src/parser/scanner.lex

```diff
@@ -228,6 +229,7 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
 {FALSE}                     { yylval->boolval = false; return TokenType::BOOL; }
 {ORDER}                     { return TokenType::KW_ORDER; }
 {INGEST}                    { return TokenType::KW_INGEST; }
+{COMPACT}                   { return TokenType::KW_COMPACTION; }
```

Comment:
I have a question. Why keyword is incomplete? Is there any other purpose?

Reply:
actual I don't know which one is correct, use "COMPACT" just for shot